core/src/display_object/avm2_button.rs

```diff
@@ -208,6 +208,9 @@ impl<'gc> Avm2Button<'gc> {
                     Ok(child) => {
                         child.set_matrix(context.gc_context, record.matrix.into());
                         child.set_depth(context.gc_context, record.depth.into());
+                        if let Some(clip) = child.as_movie_clip() {
+                            clip.set_is_button_state(context.gc_context);
```

Comment:
As currently written this flag doesn't strictly mean "this is a button state". SimpleButton groups multiple children for the same state under a parent clip that it instantiates. So that holder clip *doesn't* get the button-stage flag, but its children do, even though they're not "button states" per se. Is this correct?

Reply:
Honestly, I'm not really sure how simplebutton interacts with orphan movies - this was needed in order to keep the existing test spassing.